examples/with-tailwindcss/tailwind.config.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  purge: ['./components/**/*.js', './pages/**/*.js'],
+  purge: ['./components/**/*.js', './pages/**/*.js', './components/**/*.jsx', './pages/**/*.jsx', './components/**/*.ts', './pages/**/*.ts', './components/**/*.tsx', './pages/**/*.tsx'],
```

Comment:
I think you could potentially do this, haven't tried it though:

```suggestion
  purge: ['./components/**/*.{js,ts,jsx,tsx}', './pages/**/*.{js,ts,jsx,tsx}'],
```